### ISSUE TITLE
No events in "each"

### ISSUE THREAD

**Issue by bensmeets** [chriswessels/meteor-hammer#4]
Hi, it looks like any "gesture" (e.g. tap) is not registered if the css selector references a child within an each loop.

**Comment by chriswessels** [comment]
Hey @bensmeets,

Thanks for the issue report.

Could you provide an example?

I've just conducted the following test with success:
```javascript
if (Meteor.isClient) {
  // counter starts at 0
  Session.setDefault('counter', 0);

  Template.hello.helpers({
    counter: function () {
      return Session.get('counter');
    }
  });

  Template.hello.gestures({
    'tap .test': function (event) {
      console.log('tap .test in gestures()');
    }
  });

  Template.hello.helpers({
    looper: function () {
      return ['1', '2', '3'];
    }
  });

  Template.hello.events({
    'click button': function () {
      // increment the counter when button is clicked
      Session.set('counter', Session.get('counter') + 1);
    }
  });
}
```

```html
<head>
  <title>meteor-test-app</title>
</head>

<body>
  <h1>Welcome to Meteor!</h1>
  {{> hello}}
</body>

<template name="hello">
  <button>Click Me</button>
  <p>You've pressed the button {{counter}} times.</p>
  {{#each looper}}
    <div class="test">{{this}}</div>
  {{/each}}
</template>
```

When tapping on 1, 2 or 3 I get the expected message logged to the console.

Kind regards,
Chris

**Comment by bensmeets** [comment]
oef, hard one to track down. But I thing I've got it. You can view the non-working version here: http://meteorpad.com/pad/ejSqMRJdciKNz2oii/Tap

During this, it seems it's not because of the "each", but because of the Dependency. If instead of using deps I let the results-helper return the array directly, everything works.

**Comment by bensmeets** [comment]
I've brought the example down to a bare minimum, removing the manual dependancy to make sure that wasn't causing it. This version uses a normal collection setup. Same issue, not sure why.

http://meteorpad.com/pad/uqsjQnuNsJD7qCbqS/Tap2

**Comment by chriswessels** [comment]
Hey @bensmeets 

I had some time to look at this recently and have released 3.0.0-rc1. The package is architected slightly differently under the hood and uses a single instance of Hammer.js attached to the body element which is immune to Blaze DOM updates (which were previously causing the bug you experienced).

Hammer options are now set application-wide rather than per-template. `Template.templateName.hammerOptions(options)` has been deprecated and replaced with `$('body').data('hammer').set(options)`.

You can try the new package with
    $ meteor add chriswessels:hammer@3.0.0-rc1

Please let me know if it works for you.

Thanks,
Chris

**Comment by chriswessels** [comment]
Closing this for now as I think it's fixed with the 3.0.0-rc2 version.

Will be releasing 3.0.0 stable soon for general use.

Thanks for the issue report @bensmeets.
